README.rst

```diff
@@ -72,6 +72,7 @@ What's New?
 
 in development
 ^^^^^^^^^^^^^^
+* Make intersphinx links blue instead of red. Keep red color for internal links only.
```

Comment:
I think that the important change is that you can now, as a theme designer, you can differentiate between internal and external documentation links. 

```suggestion
* Intersphinx links have now dedicated markup.
  With the default theme,
  this allows to have the external intershinx links blue while the internal links are red.
```